README.md

```diff
@@ -1,3 +1,32 @@
-# G1ANT.Tutorials
+# G1ANT.Robot Tutorials
+
+Welcome to the open source [tutorials](https://learning.g1ant.com) for [G1ANT.Robot](https://www.g1ant.com/). Please review this README file to understand how you can assist in contributing to the G1ANT.Robot tutorials.
+
+## Getting Started
+
+Contributing to open source is more than just providing updates, it's also letting us know when there is an issue. You can submit issues with the standard [Issues](https://help.github.com/articles/about-issues/) tab on GitHub.
+
+### Prerequisites
+
+Contributing to the documentation requires a GitHub account and some [Markdown](https://guides.github.com/features/mastering-markdown/) knowledge.
+
+### Conventions
+
+Folders are treated as separated courses in the Tutorials. Folder names are used for the titles of courses, so [For Beginners](../For Beginners/) folder will turn into “For Beginners” course title on the website.
+
+Files represent individual lessons within a course. The filenames are sequential numbers determining the lessons order. We start with 10 and use the increments of 10 for the filenames, so that there is some room left for additional intermediate lessons that might appear in future and could be placed between existing ones without disturbing the whole file structure and the resulting lessons order in the Tutorials.
```

Comment:
```suggestion
Files represent individual lessons within a course. The filenames are sequential numbers determining the lessons order. We start with 010 and use three digits and the increments of 10 for the filenames, so that there is some room left for additional intermediate lessons that might appear in future and could be placed between existing ones without disturbing the whole file structure and the resulting lessons order in the Tutorials.
```